shell/platform/android/android_context_gl.cc

```diff
@@ -105,10 +109,135 @@ static bool TeardownContext(EGLDisplay display, EGLContext context) {
   return true;
 }
 
+class AndroidEGLSurfaceDamage {
+ public:
+  void init(EGLDisplay display, EGLContext context) {
```

Comment:
`context` isn't used below. was the plan to use it in the future?

Reply:
Not really. I think I just overlooked it.